### PR TITLE
Add project template prompt, support apic

### PR DESCRIPTION
**Pull request by bajtos**
When creating a new LoopBack project, let the user choose a template they would like to use.

When running from apiconnect CLI, remove "api-server" from the templates list and offer "hello-world" as the default template.

Requires https://github.com/strongloop/loopback-workspace/pull/255

/to @ritch please review
/cc @kraman

### CODE REVIEW

**Comment by bajtos** [comment]
@slnode test please

File: app/index.js [hunk 1]
```diff
@@ -64,17 +65,23 @@ module.exports = yeoman.generators.Base.extend({
...
-          // name: util.format('%s (%s)', t.name, t.description),
-          // value: t.name
-          name: t,
-          value: t
+          name: util.format('%s (%s)', t.name, t.description),
+          value: t.name
         };
       });
+
+      if (helpers.getCommandName() === 'apic') {
```
Comment (ritch) [inline app/index.js]: This is not so great. Lets add a TODO or issue to get rid of it.

**Comment by ritch** [comment]
LGTM

File: app/index.js (hunk 1 above)
Comment (bajtos) [inline app/index.js]: I think https://github.com/strongloop/generator-loopback/issues/139 will enable us to get rid of this easily. I'll leave a TODO comment.

**Pull request merged**